python/tank/bootstrap/manager.py

```diff
@@ -57,25 +60,57 @@ def __init__(self, sg_user=None):
         # defaults
         self._bundle_cache_fallback_paths = []
         self._caching_policy = self.CACHE_SPARSE
-        self._pipeline_configuration_name = None
+        self._pipeline_configuration_identifier = None
```

Comment:
Renamed from pipeline_configuration_name sense we can identify the pipeline by name or id.